### DISCUSSION TITLE
installation problems

### DISCUSSION THREAD

**Discussion by kriszdargo** [tklengyel/drakvuf#1678] [Q&A]
Hello!
I ran into quite a few problems during the installation, and I would like to ask for help with them
I tried both installation methods:
I downloaded the .deb files from the following link: https://github.com/tklengyel/drakvuf-builds/releases and installed them. The operating system I used was Ubuntu 22.04.02 (jammy), so I downloaded the corresponding .deb files.
The drakvuf bundle installed without any problems.
The Xen hypervisor, on the other hand, gave me the following error whether I tried with sudo or in root mode: 

![image](https://github.com/tklengyel/drakvuf/assets/21147086/aa5b4b8d-677b-4fa8-ae1e-8261b2ee21b6)

Then I tried the manual installation based on the guide on the website (https://drakvuf.com/).
The first error that I couldn't fix came with the make -j4 dist-xen command: 

![image](https://github.com/tklengyel/drakvuf/assets/21147086/8b798929-7e27-4278-8195-854c1c326312)

Although I already thought that this would probably be a problem in the future, I continued with the following command make -j4 dist-tools and the new errors came: 
 
![image](https://github.com/tklengyel/drakvuf/assets/21147086/6f97d53e-0064-4f05-92f1-a617e035f65a)
![image](https://github.com/tklengyel/drakvuf/assets/21147086/f3db4b26-3d1d-413a-825b-d0a97db3b3f1)
![image](https://github.com/tklengyel/drakvuf/assets/21147086/c6ab23a3-01fc-4e77-bc35-c2ab0f04106f)

 
The problems also came with the make -j4 debball command, I saw that it said during installation that pandoc is not installed, even though it was installed and the errors came at the end, the same as in the last picture.
The last error I would mention was that the deb file was not found with the dpkg -i dist/xen*.deb command.

I saw videos about DRAKVUF and it seems like a very good tool, I don't know what goes wrong with the installation.
Can I get some help?

**Comment by tklengyel** [comment]
Try to just install Xen that ships with ubuntu (apt-get install xen).
